astroquery/gemini/urlhelper.py

```diff
@@ -0,0 +1,84 @@
+"""
+Utility for building URLs for accessing the Gemini Archive RESTful services.
+
+The Gemini REST API is used behind the query calls so this utility is
+used internally by those methods to construct appropriate URLs.
+"""
+import astropy
+from astropy import units
+
+from astroquery.utils import commons
+
+
+def __handle_keyword_arg__(url, key, value):
```

Comment:
None of these functions need to be `__` - sure, they're not meant for public consumption, but the `__` is meant to hide class methods and certain special things - I don't have a comprehensive knowledge of what.  These are just regular old functions.